docs/IpInfoResultV4.md

```diff
@@ -6,7 +6,7 @@ Name | Type | Description | Notes
 **Address** | **string** |  | [optional] 
 **Geolocation** | [**IPLocation**](IPLocation.md) |  | [optional] 
 **Asn** | [**ASN**](ASN.md) |  | [optional] 
-**DataCenter** | [**DataCenter**](DataCenter.md) |  | [optional] 
+**Datacenter** | [**DataCenter**](DataCenter.md) |  | [optional] 
```

Comment:
Shouldn't the file technically be named `Datacenter.md` now? Doesn't hurt anything I guess, just for consistency sake.

Reply:
Other than that, I tested it locally a it works as expected: 
![Uploading image.png…]()